StripeUICore/StripeUICore/Source/Elements/Section/SectionContainerView.swift

```diff
@@ -198,29 +198,32 @@ extension SectionContainerView: EventHandler {
 
 extension SectionContainerView {
     class MultiElementRowView: UIView {
+        private let theme: ElementsAppearance
+        private let views: [UIView]
+
+        private var multiElementStackView: UIStackView?
```

Comment:
make this a let and non-optional or just delete it

Reply:
Removed!